### PR TITLE
[judge-d-24] renamed endpoint /contracts to /contracts/services

### PR DESCRIPTION
**Pull request by karolkalinski**
In order to increase user experience when working with API I have changed its structure. It is first part of changes towards /contracts/services/{service-name}/versions/{version}

### CODE REVIEW

File: judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java [hunk 1]
```diff
@@ -56,6 +56,16 @@ public ServiceContractsDto create(@PathVariable(name = "provider") String provid
         return serviceContractsRepository.getServiceNames();
     }
 
+    @GetMapping(value = "/contracts", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Get contracts registered", nickname = "get names of services")
```
Comment (Felipe444) [inline judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java]: Get registered contracts

File: judge-d-server/src/test/groovy/dev/hltech/dredd/interfaces/rest/contracts/ContractsControllerIT.groovy [hunk 2]
```diff
@@ -80,14 +80,26 @@ class ContractsControllerIT extends Specification {
...
         then:
             response.getStatus() == 200
             response.getContentType().contains("application/json")
             objectMapper.readValue(response.getContentAsString(), new TypeReference<List<String>>() {})
     }
 
+    def 'should calling /contracts should redirect to /contracts/services to improve api discovery'() {
+        given:
+        when:
+        def response = mockMvc.perform(
```
Comment (Felipe444) [inline judge-d-server/src/test/groovy/dev/hltech/dredd/interfaces/rest/contracts/ContractsControllerIT.groovy]: Indents here

File: judge-d-server/src/test/groovy/dev/hltech/dredd/interfaces/rest/contracts/ContractsControllerIT.groovy [hunk 3]
```diff
@@ -80,14 +80,26 @@ class ContractsControllerIT extends Specification {
...
+                get('/contracts/services')
             ).andReturn().getResponse()
         then:
             response.getStatus() == 200
             response.getContentType().contains("application/json")
             objectMapper.readValue(response.getContentAsString(), new TypeReference<List<String>>() {})
     }
 
+    def 'should calling /contracts should redirect to /contracts/services to improve api discovery'() {
+        given:
```
Comment (Felipe444) [inline judge-d-server/src/test/groovy/dev/hltech/dredd/interfaces/rest/contracts/ContractsControllerIT.groovy]: given section redundant

File: judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java (hunk 1 above)
Comment (karolkalinski) [reply judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java]: Documentation fixed

File: judge-d-server/src/test/groovy/dev/hltech/dredd/interfaces/rest/contracts/ContractsControllerIT.groovy (hunk 2 above)
Comment (karolkalinski) [reply judge-d-server/src/test/groovy/dev/hltech/dredd/interfaces/rest/contracts/ContractsControllerIT.groovy]: Code indented

File: judge-d-server/src/test/groovy/dev/hltech/dredd/interfaces/rest/contracts/ContractsControllerIT.groovy (hunk 3 above)
Comment (karolkalinski) [reply judge-d-server/src/test/groovy/dev/hltech/dredd/interfaces/rest/contracts/ContractsControllerIT.groovy]: Empty given section removed.

**Comment by karolkalinski** [comment]
Merged

**Pull request closed without merging**